common/src/main/java/com/wynntils/features/user/overlays/QuestInfoOverlayFeature.java

```diff
@@ -61,14 +62,22 @@ public void onScoreboardSegmentChange(ScoreboardSegmentAdditionEvent event) {
         }
     }
 
-    @SubscribeEvent
-    public void onTrackedQuestUpdate(TrackedQuestUpdateEvent event) {
-        if (event.getQuestInfo() == null || !autoTrackQuestCoordinates) return;
+    private static final Supplier<Location> supplier = () -> {
```

Comment:
Debatable if this should be stored here or in QuestManager. Let's move this to QuestManager and still set the compass here in the event?